br/pkg/lightning/backend/local/region_job.go

```diff
@@ -265,8 +267,6 @@ func (local *Backend) writeToTiKV(ctx context.Context, j *regionJob) error {
 	if j.regionSplitSize <= int64(config.SplitRegionSize) {
 		regionMaxSize = j.regionSplitSize * 4 / 3
 	}
-	// Set a lower flush limit to make the speed of write more smooth.
-	flushLimit := int64(writeLimiter.Limit() / 10)
```

Comment:
Why remove this logic?

Reply:
But the user may not set this field.

Reply:
default value is 16k, it's already small enough